cmd/server/main.go

```diff
@@ -1,60 +1,7 @@
 package main
```

Comment:
if we have `vela dashboard` command, we don't need this package any more

Reply:
Indeed. I still keep it as it can be the access to quickly launch API server. Let me delete it. Thanks.

Reply:
we can add a Makefile command for quick launch